packages/manager/apps/hub/public/translations/hub/Messages_fr_FR.json

```diff
@@ -2,5 +2,18 @@
   "title": "Bienvenue uapp",
   "crumb": "hub",
   "tabs_2": "Tabs 2",
-  "onboarding": "Onboarding"
+  "onboarding": "Onboarding",
+  "hub_support_title": "Dernières demandes d'assistance",
+  "hub_support_see_all": "Voir tout",
+  "hub_support_read": "Lire",
+  "hub_support_state_open": "Ouvert",
+  "hub_support_state_closed": "Fermé",
+  "hub_support_state_unknown": "Inconnu",
+  "hub_support_account_management": "Gestion du compte",
+  "hub_support_need_help": "Besoin d'aide ?",
+  "hub_support_need_help_more": "Des difficultés pour utiliser vos produits et services ? Consultez dès maintenant nos guides.",
+  "hub_support_help": "Comprendre et utiliser vos services OVHcloud",
+  "hub_support_error_message": "Impossible de récupérer vos demandes d'assistance",
+  "hub_support_error_tile_oops": "Oups ...!",
+  "hub_support_error_tile_retry": "Recharger"
```

Comment:
**nitpick:** Translations related to Support component can be created in the new namespace and load it along with the Support component.